FreeRTOS/iot-reference-nxp-rt1060
# Remove private path in project file

Description
-----------
In this PR:
* Remove private path in project file.
* Fix formatting
* Fix spelling
* Fix valid external link

I still need to define `__int64_t_defined=1` in project in order to include `PRIu64`.

Test Steps
-----------
Build this project with a new environment should not have any compilation error.

Checklist:
----------



Related Issue
-----------


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
